pyronear_ds/datasets/nasa_wildfires.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from typing import List, Optional
+
+import pandas as pd
+
+from pyronear_ds import config as cfg
+
+__all__ = ["NASAFIRMS"]
+
+
+class NASAFIRMS(pd.DataFrame):
+    """Wildfire history dataset on French territory, using data from
+    NASA satellites. Accessible by completing the form at
+    https://effis.jrc.ec.europa.eu/applications/data-request-form/
+
+    Careful when completing the form, you can either choose to get the
+    dataset in json format or xlsx format.
+    However if your source data is in a csv format, you can still use
+    this class to clean it using the parameter `fmt`.
+
+    By default, the format is considered to be json.
+
+    Args:
+        source_path: str
+            Path or URL to your version of the source data
+        use_cols: List[str]
+            List of columns to read from the source
+        fmt: str
+            Format of the source data, can either be "csv", "xlsx"
+            or "json". Default is "json".
+    """
+
+    kept_cols = [
+        "latitude",
+        "longitude",
+        "acq_date",
+        "acq_time",
+        "confidence",
+        "bright_t31",
+        "frp",
+    ]
+    fmt = "json"
+
+    def __init__(
+            self,
+            source_path: Optional[str] = None,
+            use_cols: Optional[List[str]] = None,
+            fmt: Optional[str] = None,
+    ) -> None:
+        """
+
+        Args:
+            source_path: Optional[str]
+                Path or URL to your version of the source data
+            use_cols: Optional[List[str]]
+                List of columns to keep in the dataframe
+            fmt: Optional[str]
+                Format of the source data, can either be
+                "csv", "xlsx" or "json".
+        """
+        if not isinstance(source_path, str):
+            # Download in cache
+            logging.warning(
+                f"No data source specified for {self.__class__.__name__}, trying fallback."
+            )
+            source_path = cfg.FR_NASA_FIRMS_FALLBACK
```

Comment:
Would it maybe possible to change fmt, like : 
`fmt = source_path.split('.')[-1]`
to handle files like csv

Reply:
`fmt` is an argument given by the user, I could add an else statement to return an error such as "wrong type of format, should be json, csv or xlsx", that could be better, in case of the user does not pass a permitted format

Reply:
Would like something more "automatic" where the user does not give the format but where the format is automatically guessed ? If it's the case, I'll switch to what you are proposing and modify the docstring accordingly

Reply:
Ah yes I see, so my problem is when I launch from the command line, like for me it's like I should launch like :
`python3 example_NASA_FIRMS.py /home/camille/Downloads/fire_nrt_M6_161662.csv [COLUMNS_I_SHOULD_USE] 'csv'`
But I feel like if we want to make the user really passes the `fmt` as an argument, it should be in second position. Like I like the default columns that are passed, I don't want to have to pass the list of columns I want to get everytime I'm using a different type of file. I'd like to just pass : 
`python3 example_NASA_FIRMS.py /home/camille/Downloads/fire_nrt_M6_161662.csv 'csv'`
Maybe that's because I have habits, I don't know if there is some way of changing the order you pass args in python from the command line, if there is, feel free to tell me haha

Reply:
Ooooh non you're right !! Don't know if we can change the order of the args directly from the command line, therefore I changed it in the script. I also added an else statement in case of. Now you should be able to use directly `python3 example_NASA_FIRMS.py /home/camille/Downloads/fire_nrt_M6_161662.csv 'csv'`